server/dredd.yml

```diff
@@ -0,0 +1,34 @@
+dry-run: null
+hookfiles: ./spec/unit/controllers/index.hook.js
+language: nodejs
+sandbox: false
+server: yarn dev-test
```

Comment:
Dredd starts separate dev server with `testing db`. That's needed in order not to operate with `development db`. 
Ideally, we would like to run one testing dev server for both dredd and jasmine (that should be implemented in the future)

We can't use development db because we'll probably mess it with testing records, that's why a testing server is up